03_operadores/test/test_desafio.rb

```diff
@@ -3,34 +3,64 @@
 # Desafio
 # Vale 1.5 pontos válido se criar PR até 17/08/2022 23:59:59
 #
-# Implementar o método #calculadora
+# Implementar o método #calculator
 # Adicionar assert_equal para cada método de teste com seu respectivo operador
 
-class TestDesafio < Minitest::Test
+class TestChallenge < Minitest::Test
 
-  def calculadora(valor_a, valor_b, operador)
-    return nil
+  def calculator(value_a, value_b, operator)
+    if value_a.kind_of?(Numeric) && value_b.kind_of?(Numeric)
+      if operator == "+"
+        return value_a + value_b
+      elsif operator == "-"
+        return value_a - value_b
+      elsif operator == "*"
+        return value_a * value_b
+      elsif operator == "**"
+        return value_a ** value_b
+      elsif operator == "/" && value_b != 0
+        return value_a / value_b.to_f
+      elsif operator == "%"
+        return value_a % value_b
+      elsif operator == "="
+        return value_a == value_b
+      end
+    end
+
+    "Invalid Operation"
   end
 
-  def test_soma
-    assert_equal 2, calculadora(1, 1, '+')
+  def test_sum
+    assert_equal 2, calculator(1, 1, '+')
+    assert_equal "Invalid Operation", calculator('teste', 4, '+')
   end
 
-  def test_subtracao
+  def test_subtraction
+    assert_equal 5, calculator(8, 3, '-')
+    assert_equal "Invalid Operation", calculator(15, 10, '--')
   end
 
-  def test_multiplicacao
+  def test_multiplication
+    assert_equal 12, calculator(6, 2, '*')
+    assert_equal "Invalid Operation", calculator(true, 6, '*')
   end
 
-  def test_potencia
+  def test_power
+    assert_equal 8, calculator(2, 3, '**')
+    assert_equal "Invalid Operation", calculator(nil, 7, '**')
   end
 
-  def test_divisao
+  def test_division
+    assert_equal 2.5, calculator(5, 2, '/')
+    assert_equal "Invalid Operation", calculator(20, 0, '/')
   end
 
   def test_modulo
+    assert_equal 1, calculator(9, 2, '%')
   end
 
-  def test_comparacao
+  def test_comparison
+    assert_equal true, calculator(1, 1, '=')
+    assert_equal false, calculator(1, 2, '=')
```

Comment:
[Fix]

_English_

The test is well implemented, but this line is with the wrong sign, try changing to "==" 

_Português_

O teste está bem implementado, mas essa linha está com o sinal errado, tente alterar para "=="
```suggestion
    assert_equal false, calculator(1, 2, '==')
```

Reply:
English:
The choice of using a single equal sign (=) instead of double equal sign (==) was a project strategy, since the "calculator" method implements a feature designed at the common user, without prior knowledge about programming, which use, normally, a single equal sign.

Português:
A opção pelo uso de um único sinal de igual (=) em vez de duplo sinal de igual (==) foi uma estratégia de projeto, pois o método "calculadora" implementa uma funcionalidade projetada para o usuário comum, sem conhecimento prévio de programação, que utiliza, normalmente, um único sinal de igual.

Reply:
_English_

I nice catch by the team, didn't realize until seeing the calculator method with more attention.

_Português_

Uma boa sacada do time, não havia percebido até ver o método da calculadora com mais atenção.